client/components/app.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import axios from 'axios';
-import DescriptionInfoList from './descriptionInfoList.jsx';
 import News from './news.jsx';
 import styled from 'styled-components';
+import Description from './description.jsx';
```

Comment:
I would recommend capitalized file names (for React components) as per airbnb style guide https://github.com/airbnb/javascript/tree/master/react#naming